src/basic/parse-util.h

```diff
@@ -11,6 +11,26 @@
 
 typedef unsigned long loadavg_t;
 
+typedef enum PcrNumber {
```

Comment:
This defines names, so PcrName would be better

Reply:
or just Pcr

Reply:
> or just Pcr

Oh.. okay sir.
Thank you for giving so much time today.